SingularityService/src/main/java/com/hubspot/singularity/data/RackManager.java

```diff
@@ -39,33 +43,41 @@ protected String getRoot() {
   }
 
   public void activateLeaderCache() {
-    leaderCache.cacheRacks(getObjects());
+    leaderCache.cacheRacks(getObjectsNoCache(getRoot()));
   }
 
-  public Optional<SingularityRack> getRack(String rackName) {
+  @Override
+  public Optional<SingularityRack> getObjectFromLeaderCache(String rackId) {
     if (leaderCache.active()) {
-      return leaderCache.getRack(rackName);
+      return leaderCache.getRack(rackId);
```

Comment:
We're using both `rackName` (in `SingularityLeaderCache`) and `rackId`. We should pick one for consistency

Reply:
consolidating on rackId 👍